src/nls/ja/strings.json

```diff
@@ -237,7 +237,7 @@
         "SECTION_COLLAPSE": ": ダブルクリックして折りたたむ",
         "SECTION_EXPAND": ": ダブルクリックして展開",
         "PANEL_COLUMN_COLLAPSE": ": クリックして折りたたむ",
-        "PANEL_COLUMN_EXPAND": ": クリックして展開",
+        "PANEL_COLUMN_EXPAND": " : クリックして展開",
```

Comment:
I sent mail to iDev to see why they update the translation.

Feng

From: Joel Brandt <notifications@github.com<mailto:notifications@github.com>>
Reply-To: adobe-photoshop/spaces-design <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Monday, October 26, 2015 at 3:12 PM
To: adobe-photoshop/spaces-design <spaces-design@noreply.github.com<mailto:spaces-design@noreply.github.com>>
Cc: fzhu <fzhu@adobe.com<mailto:fzhu@adobe.com>>
Subject: Re: [spaces-design] Updated by ALF automation. (#3135)


In src/nls/ja/strings.json<https://github.com/adobe-photoshop/spaces-design/pull/3135#discussion_r43060870>:

> @@ -237,7 +237,7 @@
>          "SECTION_COLLAPSE": ": $B%@%V%k%/%j%C%/$7$F@^$j$?$?$`(B",
>          "SECTION_EXPAND": ": $B%@%V%k%/%j%C%/$7$FE83+(B",
>          "PANEL_COLUMN_COLLAPSE": ": $B%/%j%C%/$7$F@^$j$?$?$`(B",
> -        "PANEL_COLUMN_EXPAND": ": $B%/%j%C%/$7$FE83+(B",
> +        "PANEL_COLUMN_EXPAND": " : $B%/%j%C%/$7$FE83+(B",


This doesn't seem like a whitespace problem -- it seems like an "insertion of the replacement string" problem. I can't imagine the : ' should be in the string at all (for lines 237-240, not just 240). It looks like somehow the":` that closes the key is getting copied to the value.

Just my $0.02...

-
Reply to this email directly or view it on GitHub<https://github.com/adobe-photoshop/spaces-design/pull/3135/files#r43060870>.